manifests/params.pp

```diff
@@ -41,6 +41,9 @@
   $fork_remote_name = undef
   $upstream_remote_name = 'upstream'
 
+  $foreman_scm_revision = 'develop'
+  $katello_scm_revision = 'master'
```

Comment:
Shouldn't these be 3.18 and whatever Foreman matches that? (2.3?) That way the defaults are correct if you check out the branch.

Reply:
They get overridden by our packer scripting via installer flags, I don't think anyone is directly checking these branches out ... yet